package.json

```diff
@@ -69,11 +71,15 @@
     "regenerator-runtime": "^0.13.5",
     "rollup": "^1.27.12",
     "rollup-plugin-typescript2": "^0.27.0",
+    "styled-components": "4.4.1",
+    "styled-system": "^5.1.5",
```

Comment:
since they are already added as `peerDependencies` I think they should be removed from dependencies.

Reply:
According to `styled-components` [documentation](https://styled-components.com/docs/faqs#i-am-a-library-author-should-i-bundle-styledcomponents-with-my-library), it needs to be a devDependency as well as a peerDependency.  As for `styled-system` it needs to be a devDependency too. Otherwise, the module cannot be found when running storybook.